Libplanet/Net/Swarm.cs

```diff
@@ -801,6 +805,17 @@ private async Task RefreshAllocate()
             }
         }
 
+        private async Task RefreshPermissions()
+        {
+            TimeSpan lifetime = TurnPermissionLifetime;
+            while (Running)
+            {
+                await Task.Delay(lifetime - TimeSpan.FromMinutes(1));
+                await Task.WhenAll(
+                    _peers.Keys.Select(p => CreatePermission(p)));
```

Comment:
```suggestion
                    _peers.Keys.Select(CreatePermission));
```

It's trivial though.